packages/mui-material/src/Stepper/Stepper.test.tsx

```diff
@@ -5,7 +5,7 @@ import Step, { StepProps, stepClasses } from '@mui/material/Step';
 import StepLabel from '@mui/material/StepLabel';
 import StepConnector, { stepConnectorClasses } from '@mui/material/StepConnector';
 import StepContent, { stepContentClasses } from '@mui/material/StepContent';
-import Stepper, { stepperClasses as classes } from '@mui/material/Stepper';
+import Stepper, { stepperClasses as classes, stepperClasses } from '@mui/material/Stepper';
```

Comment:
`stepperClasses` is already imported and named as `classes`, so you can use `classes` instead of importing `stepperClasses` again